tests/unit/synapseclient/unit_test_client.py

```diff
@@ -263,6 +263,12 @@ def setup(self):
             'etag': 'etag',
             'name': 'entity name'
         }
+        self.docker = {
+            'versionNumber': 7,
```

Comment:
docker is not version-able entity. The test should ensure that without versionNumber in a mock docker entity, the function still executed as expected.